packages/sit-onyx/src/components/OnyxTooltip/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { OnyxColor } from "../../types";
+
+export type OnyxTooltipProps = {
+  /**
+   * Text to display inside the tooltip.
+   */
+  text: string;
+  /**
+   * Optional icon to show on the left of the text.
+   */
+  icon?: string;
+  color?: Extract<OnyxColor, "neutral" | "danger">;
+  /**
+   * How to position the tooltip relative to the parent element.
+   */
+  position?: TooltipPosition;
+  /**
+   * If `true`, the tooltip will use the full width of the parent/slot element.
```

Comment:
```suggestion
   * If `true`, the tooltip will match the width of the parent/slot element.
```
because it goes both ways, right?: even when the parent is less wide, the tooltip will be restricted to that width